jsonmap.go

```diff
@@ -564,15 +564,15 @@ type toStringable interface {
 }
 
 // This is a horrible hack of the go type system
```

Comment:
small nit: the doc string is snarky and could instead explain the use case, reference OpenAPI spec, etc